extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/sql_load_script/InvalidMultilineSqlLoadScriptTestCase.java

```diff
@@ -5,22 +5,20 @@
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
 import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.RegisterExtension;
 
 import io.quarkus.hibernate.orm.MyEntity;
 import io.quarkus.test.QuarkusUnitTest;
 
 public class InvalidMultilineSqlLoadScriptTestCase {
-    @RegisterExtension
+    //@RegisterExtension
```

Comment:
May be we should uncomment after https://github.com/quarkusio/quarkus/pull/4252 was merged?